src/Interface_TEST.cc

```diff
@@ -166,7 +166,7 @@ TEST(Interface, FetchResources)
   {
     // Check it's not cached
     common::URI worldUrl{
-      "https://fuel.ignitionrobotics.org/1.0/nate/worlds/Empty"};
+      "https://fuel.ignitionrobotics.org/1.0/openrobotics/worlds/Test world"};
```

Comment:
All of the `openrobotics` should be `OpenRobotics` (camelcase).

Reply:
That would make the tests fail on macOS, see https://github.com/ignitionrobotics/ign-fuel-tools/issues/80. The lowercase tests seem to be working on both Linux and Mac.

Reply:
I'll merge this as it is, we can address this on blueprint and merge it forward when possible.